src/collections/_documentation/apm/apm-glossary.md

```diff
@@ -0,0 +1,164 @@
+
+### Properties
+
+Each span has a unique identifier kept in the `span_id` attribute.
+
+Start time and end time are stored in the `start_timestamp` and `timestamp` attributes, respectively. The format is an RFC 3339 / ISO 8601-compatible string or a numeric value with the number of seconds since the UNIX Epoch.
+
+The name of the operation is stored in the “op” parameter. Examples are `http`, `sql.query`, `redis.command`.
+
+Additional data about the operation can be stored in the `data` and `tags` attributes.
+
+## Trace
+
+A trace is a collection of spans that have the same `trace_id` attribute. Traces consist of all spans generated by the distributed system during the processing of the input data. Traces are not explicitly started or finished. The start and end of a trace are defined by the transactions and spans participating in the trace.
+
+Traces that start with the same set of API endpoint calls can potentially differ in terms of what code paths were executed. For example, an initial trace may result in a cache miss at the Cache Service, and a subsequent trace may include more information instrumented at the Cache Service. Although these two traces are similar, they are not identical.
+
+### Propagation
+
+Processing of the input data in the distributed environment means that an arbitrary number of the systems/microservices can be involved.
+
+To connect all the spans that were created during the processing of the input data, we need to be able to propagate the basic identifier. By forwarding `trace_id` and `parent_span_id` as data travels through various applications in a microservice ecosystem, we can reconstruct data flows later.
+
+Trace metadata is propagated through http headers that are included in the http request to the external system.
+
+## Transaction
+
+Transaction is an event attribute that provides initial context to the error report. This makes it possible for the error to be linked to a specific part of the system. An example of the transaction name can be “/users/” which is the name for the endpoint that received the request that failed during the processing.
+
+With tracing, the transaction is a parent span to all spans created during the processing of the request in the single server.
+
+Once the request is processed, the transaction is sent together with all child spans to Sentry.
+
+### Event Types
+
+The SDKs already collect a lot of information about the processing of the request. In the case of an error, the SDK attaches the info to the error report to provide rich context of what happened before the error appeared -- information like tags and breadcrumbs.
+
+If the error doesn’t appear during the processing of the request, we discard the context data once processing is complete.
+
+We want to leverage this context collection to build spans, and once a request is finished, we want to send the data to Sentry.
```

Comment:
Why are these paragraphs here? The payload below gives an example of a transaction event, but no other event types are covered. If we don't reference this anywhere could we get by without it?

Reply:
Yeah, I can remove this part. I think this was part of the forum glossary. https://forum.sentry.io/t/sentrys-apm-docs-alpha/7843

Reply:
@markstory to be clear, we should remove lines 131-133?

Reply:
Unless we need a place to refer people to on the shape of a transaction event, I would remove 127-164. If we do need a reference transaction event, then remove 129-134.